src/routes/index.tsx

```diff
@@ -0,0 +1,12 @@
+import type { RouteObject } from "react-router-dom"
```

Comment:
Same as the suggestion above

```suggestion
import type { RouteObject } from 'react-router-dom'
```